### PR TITLE
FIX: Feeds revision

### LINKED ISSUE #2275: Taichung feed seems dicontinued
The feed URL https://www.tc.edu.tw/net/netflow/lkout/recent/ returns 404. The website has completely changed (I remember the old looks). It is currently available only in Chinese, I tried searching for the feed using google translator but didn't find anything. The button for English language redirects to a completely different site.

I think we can safely assume the feed is dead.

### PR DESCRIPTION
**Pull request by gethvi**
This pull request offers a revision of some feeds and their configuration.

Also fixes #2275.

### CODE REVIEW

**Comment by aaronkaplan** [comment]
Hi @gethvi , should this PR continue and make it to 3.1? If so, ... we plan to release 3.1 by this FR. Or should I move that to 3.2?
PS: as always (and implicitly given, but nevertheless it's important for me to say it explicitly): thanks (!!!) for your contributions.

**Comment by gethvi** [comment]
Should be ready.

File: intelmq/lib/upgrades.py [hunk 1]
```diff
@@ -850,6 +864,24 @@ def v310_feed_changes(configuration, harmonization, dry_run, **kwargs):
...
+        messages.append('The feed "Viriback Unsafe Site" has been replaced. Please see the feed'
+                        ' "Viriback C2 Tracker" and'
+                        ' adjust your configuration. Affected bots: %s' % ', '.join(found_viriback))
+
+    if found_netlab_mirai_scanner:
+        messages.append('A discontinued feed "Netlab Mirai Scanner" has been found '
+                        'as bot %s.' % ', '.join(sorted(found_netlab_mirai_scanner)))
+
+    if found_benkow_panels:
+        messages.append('The feed "Benkow Malware Panels Tracker" has been changed. Please see the feed\'s'
```
Comment (sebix) [inline intelmq/lib/upgrades.py:882]: As in #2268 I'd prefer if the configuration upgrade command would actually upgrade the configuration to a working state, but I won't block here.

**Comment by aaronkaplan** [comment]
We have some merge conflicts. Pls. review @gethvi and @sebix , if possible for you. Thanks a lot!

**Comment by gethvi** [comment]
> We have some merge conflicts. Pls. review @gethvi and @sebix , if possible for you. Thanks a lot!

Should be resolved. :)

**Comment by aaronkaplan** [comment]
damn again a merge conflict... sorry. I think that's because I merged in the other PR first.

**Comment by gethvi** [comment]
> damn again a merge conflict... sorry. I think that's because I merged in the other PR first.

Resolved again. :rocket:

**Pull request merged**